addons/mrp/views/stock_orderpoint_views.xml

```diff
@@ -7,7 +7,7 @@
         <field name="arch" type="xml">
             <field name="route_id" position="after">
                 <field name="show_bom" column_invisible="True"/>
-                <field name="bom_id" optional="hide" invisible="not show_bom" context="{'default_product_tmpl_id': product_tmpl_id}"/>
+                <field name="bom_id" optional="hide" invisible="not show_bom" context="{'default_product_tmpl_id': product_tmpl_id, 'from_orderpoint': True}"/>
```

Comment:
Since the point is to display the quantity/units of the bom, I'd rather have it a more explicit name for that purpose, so that we could use it elsewhere should the need arise.